mail/urls.py

```diff
@@ -19,5 +20,6 @@
     url(r'^api/v0/mail/search/$', SearchResultMailView.as_view(), name='search_result_mail_api'),
     url(r'^api/v0/mail/course/(?P<course_id>[\d]+)/$', CourseTeamMailView.as_view(), name='course_team_mail_api'),
     url(r'^api/v0/mail/learner/(?P<student_id>[\d]+)/$', LearnerMailView.as_view(), name='learner_mail_api'),
+    url(r'^api/v0/mail/bounce_event/$', EmailBouncedView.as_view(), name='email_bounced_view'),
```

Comment:
Please change this endpoint to `^api/v0/mail/webhook/$` so we can handle other non-bounce events in the future if necessary. Also change `email_bounced_view` to `mail-webhook`